demos/showcase/src/app/app.component.scss

```diff
@@ -44,3 +44,9 @@ pxb-scorecard {
     }
     margin-right: 16px;
 }
+
+.test-background-image .pxb-drawer-header {
+    background-image: url('../assets/topology_40.png') !important;
+    background-color: #deadbeef !important;
+    background-size: cover !important;
```

Comment:
are the importants necessary? My hope is that users will be able to customize their implementation of our components without having to slap `!important` all over their stylesheets.

Reply:
I was able to get rid of the !important. This CSS is directly styling the mat-toolbar so I just needed a selector with higher specificity.